assets/js/app.js

```diff
@@ -69,6 +69,25 @@ Hooks.HandleUpdateOrder = {
   }
 }
 
+Hooks.Accordion = {
```

Comment:
accordians can be implemented with no JS https://www.w3schools.com/howto/howto_js_accordion.asp

Reply:
@dgross881 we have this already but the issue was the accordion is inside a form that validates on change so it doesnt retain the state when DOM patching

Reply:
@rcmonsayac thanks and noted